api/all/src/main/java/io/opentelemetry/api/metrics/MeterBuilder.java

```diff
@@ -5,19 +5,12 @@
 
 package io.opentelemetry.api.metrics;
 
-/**
- * Builder class for creating {@link Meter} instances.
- *
- * @since 1.4.0
```

Comment:
Maybe this is a bigger discussion, but what's our current stance on `@since <version>` strings? I know there was an OLD issue to normalize it across the repo, but I've long lost track of it.

Reply:
I made the same observation but forgot to submit the comment:

I removed these @since annotations because they don't reflect when the metrics API was published. If we're interested in keeping this, we should: 1. Add it at the class level for each public class in the metrics package. 2. Be more consistent about ensuring this is added to new APIs.

Reply:
Our original plan was to assume everything was `@since 1.0.0`, and then only add explicit `@since` tags when we added new APIs. I think we have not done this consistently or well so far, though.